wiki/osu!_originals/fr.md

```diff
@@ -623,6 +641,15 @@ Ces musiques ont été [annoncées](https://osu.ppy.sh/home/news/2023-02-01-twin
 | [takehirotei as "Infinite Limit" - C:// YOU NEX-TIME](https://soundcloud.com/takehirotei/takehirotei-as-infinite-limit-see-you-next-time) | [#1](https://osu.ppy.sh/beatmapsets/2028305) | Tiebreaker des Grandes Finales |
 | [takehirotei - See you next time!](https://soundcloud.com/takehirotei/see-you-next-time) |  | Thème du générique de fin |
 
+#### ![](/wiki/shared/mode/mania.png) osu!mania 4K Chinese National Cup 2023
+
+| Musique | Beatmap | Notes |
+| :-: | :-: | :-- |
+| dMinor117 & ShirinNeko - d Minor Sonatine (Insomnia) | [#1](https://osu.ppy.sh/beatmapsets/2019496) | Playoff 2 RC5 |
```

Comment:
any french reviewer for this one? (@DoryanR)

Reply:
It's good for me 👍